src/main/java/org/entando/kubernetes/model/bundle/processor/DirectoryProcessor.java

```diff
@@ -109,15 +110,18 @@ public Reportable getReportable(BundleReader bundleReader, ComponentProcessor<?>
         List<String> idList = new ArrayList<>();
 
         try {
-            final String componentFolder = "/" + bundleReader.getBundleCode();
-            idList.add(componentFolder);
+            final String resourceFolder = BundleUtilities.determineBundleResourceRootFolder(bundleReader);
+            if (! resourceFolder.equals("/")) {
+                idList.add(resourceFolder);
+            }
 
             List<String> resourceFolders = bundleReader.getResourceFolders().stream().sorted()
                     .collect(Collectors.toList());
-            for (final String resourceFolder : resourceFolders) {
+            for (final String resourceDir : resourceFolders) {
```

Comment:
I don't get the rename from **resourceFolder** to **resourceDir**

you wanted to avoid confusion with resourceFolders?

Reply:
Because I'm using resouceFolder as var name in the step at line 113, that in my opinion is more important that has a meaningful name rather than the one in a for loop